### PR TITLE
Change cmake type of source dir variable name

### PR DESCRIPTION
**Pull request by szledan**
See more: https://cmake.org/cmake/help/v3.0/command/project.html#command:project

### CODE REVIEW

**Comment by elecro** [comment]
This PR contains two commits and only one of them is the cmake source dir thingy. Is that intentional?

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,5 @@
+BSD 2-Clause License
+
 /* Copyright (C) 2015-2016, Gepard Graphics
```
Comment (kkristof) [inline LICENSE]: you might want to bump the year

Comment (kkristof) [reply LICENSE]: but this can be leaved to a different PR, +1 LGTM

**Comment by szledan** [comment]
Hi @elecro !
Yes, the first commit, which added the license name is intentional committed. But you are right, I forgot mention in PR.
I will separate independent changes in to different PRs in the future.

**Pull request merged**